ui/components/src/ui-components/propertygrid/component/VirtualizedPropertyGrid.tsx

```diff
@@ -397,7 +392,10 @@ const FlatGridItemNode = React.memo(
                 style={gridContext.style}
                 category={node.derivedCategory}
                 onExpansionToggled={onExpansionToggled}
-                highlightedPropertyProps={gridContext.highlightedPropertyProps}
+                highlight={gridContext.highlight ?
```

Comment:
Not sure I understand this. Why do we need `applyOnCategory` prop on the Category and pass false? Why not instead just have plain `HighlightingComponentProps` in category props and not pass them if `!gridContext.highlight.filteredTypes.includes(FilteredType.Category)`?

Reply:
Just thought that it would be pettier to implement passing highlight on category and record the same way. But it doesn't really make sense. Changed it to the suggested way.